velox/docs/functions/spark/math.rst

```diff
@@ -108,11 +108,20 @@ Mathematical Functions
 
 .. spark:function:: rand() -> double
 
-    Returns a random value with independent and identically distributed uniformly distributed values in [0, 1). ::
+    Returns a random value with independent and identically distributed uniformly distributed values in [0, 1).
+    Its alias is ``random``.
```

Comment:
A convention is to document this as part of 'random' function which should say "An alias for 'rand'." Let's remove this sentence from here.